ddl/backfilling_scheduler.go

```diff
@@ -311,8 +328,21 @@ func (b *ingestBackfillScheduler) close(force bool) {
 		return
 	}
 	close(b.taskCh)
-	b.copReqSenderPool.close(force)
-	b.writerPool.ReleaseAndWait()
+	if b.copReqSenderPool != nil {
```

Comment:
if we first cancel writerPool， then colse readerPool more safe？

Reply:
Why? If we cancel the `writerPool` first, the chunk read by the reader will be blocked.